### DISCUSSION TITLE
login problem

### DISCUSSION THREAD

**Discussion by soxft** [community/community#20079] [Copilot Conversations]
I am using Visual Studio Code, After I have logged in, I can use copilot normally. But every time I restart the app or reload, copilot requires me to log in again

**Comment by JaimeTR** [comment]
I installed GitHub Copilot Nightly, and it fixed the problem for me.
Before installing GitHub Copilot Nightly, please uninstall or disable Github copilot and install the extension from https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly

If this answer is helpful, please 'Mark as answer'. :)

**Reply by soxft** [reply]
It still doesn't work on my device, maybe it's because I installed the beta version of the macos system, or it's caused by other extensions
